src/riak_core_info_service_process.erl

```diff
@@ -0,0 +1,69 @@
+-module(riak_core_info_service_process).
+-behaviour(gen_server).
+
+-record(state, {
+    source = undefined :: riak_core_info_service:callback(),
```

Comment:
Suggest a different name; I first thought `source` was the source of the request. Perhaps `service_provider`?